exchanges/gateio/gateio_wrapper.go

```diff
@@ -2384,6 +2360,19 @@ func (g *Gateio) WebsocketSubmitOrder(ctx context.Context, s *order.Submit) (*or
 	}
 }
 
```

Comment:
🚧  [This thread is outstanding](https://github.com/thrasher-corp/gocryptotrader/pull/1382#discussion_r2099599344)

* 🚧 Understood; But I think you misunderstood what I'm asking for:
> Can you add a comment
```suggestion
// timeInForceString returns the most relevant time-in-force exchange string for a TimeInForce
// Any TIF value that is combined with POC, IOC or FOK will just return that
// Otherwise the lowercase representation is returned
```

The point of a comment like this is to make sure that the *intention* is decoupled from the implementation. So if there was a bug, someone know what we're trying to achieve. And so they can refactor it whilst following the intentions.

* But I'm left wondering: Isn't the only remaining valid value `gctTIF` ?
So yeah, for the sake of simplicity and not leaving the reader wondering, maybe just put that in there.
And if you use a case for that, I wouldn't mind having a default that `panics`, since it should be unreachable.